src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FormattingLanguageServerClient.cs

```diff
@@ -154,12 +206,37 @@ private RazorDocumentRangeFormattingResponse Format(RazorDocumentRangeFormatting
                 var baselineOutputText = SourceText.From(baselineOutputHtml);
                 var changes = SourceTextDiffer.GetMinimalTextChanges(baselineInputText, baselineOutputText, lineDiffOnly: false);
                 var edits = changes.Select(c => c.AsTextEdit(baselineInputText)).ToArray();
+#if false // Use the results from the real formatter
```

Comment:
Is there harm in removing all this magic now that we can invoke the real one? Even if it is through reflection magic? 😄

Reply:
It depends entirely on the outcome of https://github.com/dotnet/aspnetcore-tooling/pull/3061#discussion_r569679668